File: Shinhan/Programmers/Day7/Query.java
class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        for (int[] query : queries) {
            int s = query[0];
            int e = query[1];
            int k = query[2];

            for (int i = s; i <= e; i++) {
                if (i % k == 0) {
                    arr[i]++;
                }
            }
        }
        return arr;
    }
}

File: Shinhan/Programmers/Day14/Query.js
function solution(arr, queries) {
    var answer = [];
    
    for(let i = 0; i<queries.length; i++){
        var start = queries[i][0];
        var end = queries[i][1];
        for(let j = start; j<=end; j++){
            arr[j] += 1;
        } 
    }
    return arr;
}